build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+    id("com.exactpro.th2.gradle.publish") version "0.0.4"
```

Comment:
note: There is a newer version of th2 gradle plugin 0.0.5 with the latest BOM updates